recipe/meta.yaml

```diff
@@ -1,25 +1,26 @@
 {% set name = "aiohttp-jinja2" %}
-{% set version = "1.4.2" %}
+{% set version = "1.5" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9c22a0e48e3b277fc145c67dd8c3b8f609dab36bce9eb337f70dfe716663c9a0
+  sha256: 7c3ba5eac060b691f4e50534af2d79fca2a75712ebd2b25e6fcb1295859f910b
 
 build:
   number: 0
   noarch: python
+  skip: True  # [py<35]
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
 
 requirements:
   host:
     - aiohttp >=3.2.0
     - jinja2 >=2.10.1
     - pip
```

Comment:
- Missing wheel package in requirements
- Missing setuptools package in requirements

Reply:
Added wheel and setuptools:

https://github.com/AnacondaRecipes/aiohttp-jinja2-feedstock/blob/update_to_1.5/recipe/meta.yaml#L22&L23